rslib/src/stats/card.rs

```diff
@@ -66,10 +71,15 @@ impl Collection {
             ),
             CardQueue::Review | CardQueue::DayLearn => (
                 {
-                    let days_remaining = due - (self.timing_today()?.days_elapsed as i32);
-                    let mut due = TimestampSecs::now();
-                    due.0 += (days_remaining as i64) * 86_400;
-                    Some(pb::generic::Int64 { val: due.0 })
+                    if card.ctype == CardType::New && card.original_due != 0 {
```

Comment:
```suggestion
                    if card.ctype == CardType::New {
```

I think you're right that it's not necessary - it shouldn't do any harm, but might as well remove it.